### PR TITLE
Add CoreCLR projects to the Roslyn solutions

### LINKED ISSUE #3876: Add the CoreCsc and CoreVbc to the build solutions when the VS hang in the DNX build tools is resolved
The VS hang is currently blocking including these projects in any dev solution. See here for more details: https://github.com/dotnet/buildtools/issues/191

### PR DESCRIPTION
**Pull request by agocke**
Update the DNX build tools, which fixes #3876. Now that the
bug is resolved, add the CoreCLR projects directly to the
compiler solutions.

@jaredpar

### CODE REVIEW

**Comment by agocke** [comment]
retest this please

File: Roslyn.sln [hunk 1]
```diff
@@ -5692,6 +5696,86 @@ Global
...
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Any CPU.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Any CPU.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|ARM.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|ARM.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Mixed Platforms.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|Mixed Platforms.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x64.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x64.Build.0 = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x86.ActiveCfg = Release|Any CPU
+		{E3CD2895-76A8-4D11-A316-EA67CB5EA42C}.DogfoodRelease|x86.Build.0 = Release|Any CPU
```
Comment (jaredpar) [inline Roslyn.sln]: The DogfoodDebug and DogfoodRelease entries look new.  Is there a reason that we added them?

File: build/Targets/VSL.Imports.targets [hunk 2]
```diff
@@ -165,7 +165,7 @@
 
  </PropertyGroup>
 
-  <Import Condition="'$(TestCoreClr)' == 'True'" Project="$(BuildToolsDir)Build.Common.Targets" />
+  <Import Condition="'$(TargetCoreClr)' == 'True'" Project="$(BuildToolsDir)Build.Common.Targets" />
```
Comment (jaredpar) [inline build/Targets/VSL.Imports.targets]: Yay for better names!

**Comment by jaredpar** [comment]
:+1:

File: Roslyn.sln (hunk 1 above)
Comment (agocke) [inline Roslyn.sln]: Those are actually really old configurations... and I believe the only thing I did was "Add existing project" in each solution. I'm confused. I'll try this again.

**Comment by agocke** [comment]
retest this please

**Comment by agocke** [comment]
retest this please

**Pull request merged**